components/transactions/TransactionItem.js

```diff
@@ -305,6 +336,7 @@ TransactionItem.propTypes = {
     id: PropTypes.string,
     uuid: PropTypes.string,
     type: PropTypes.string,
+    kind: PropTypes.string,
```

Comment:
Maybe a slightly stricter enforcement of the `PropTypes` is;

```suggestion
    kind: PropTypes.oneOf(Object.values(TransactionKind))
```